### DISCUSSION TITLE
Pre-populated WHERE query in table "Data" view

### DISCUSSION THREAD

**Discussion by Reprevise** [dbeaver/dbeaver#35949] [Q&A] [answered]
Every time I open a specific table and go to it's Data tab, it has filters pre-applied to it. I've turned off all of the Auto-save options and restarted DBeaver but it still happens.

**Comment by TB-HunBoka** [comment, marked as answer]
Did you try to "save filter setting" manually after removing the filter?
![kép](https://github.com/user-attachments/assets/36418f0b-3655-4fb6-be51-87c2baf58c7c)

**Reply by Reprevise** [reply]
Ah, thanks I was looking for something like that.
